src/ansys/aedt/core/modeler/cad/primitives.py

```diff
@@ -1481,6 +1481,58 @@ def cover_faces(self, assignment):
         self.oeditor.CoverSurfaces(["NAME:Selections", "Selections:=", obj_to_cover, "NewPartsModelFlag:=", "Model"])
         return True
 
+    @pyaedt_function_handler(selection="assignment")
+    def uncover_faces(self, assignment):
+        """Uncover surfaces/faces.
+
+        Parameters
+        ----------
+        assignment : list
+            List of Sheet object to uncover.
```

Comment:
```suggestion
            Sheet objects to uncover.
```

If I am not wrong, this method support both, faces IDs or face object